### PR TITLE
Fix tracing state management.

### PR DESCRIPTION
**Pull request by polina-c**
Fixes https://github.com/flutter/devtools/issues/5013

Store tracing controller in memory controller, so that it is preserved after switching to other screen.

### CODE REVIEW

**Comment by bkonyi** [comment]
What's being fixed here?

**Comment by polina-c** [comment]
> What's being fixed here?

Added description

File: packages/devtools_app/lib/src/screens/memory/memory_controller.dart [hunk 1]
```diff
@@ -68,8 +69,9 @@ class MemoryController extends DisposableController
   /// The controller is late to enable test injection.
   late final DiffPaneController diffPaneController;
 
-  /// Controller for [AllocationProfileTableView].
-  final allocationProfileController = ProfilePaneController();
+  final profilePaneController = ProfilePaneController();
+
+  final tracingPaneController = TracingPaneController();
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/memory/memory_controller.dart:74]: now that we are storing this in the controller, does this need to be cleared or disposed at any point by the memory controller?

**Review by kenzieschmoll — APPROVED** [review]
one question to double check, then lgtm

File: packages/devtools_app/lib/src/screens/memory/memory_controller.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/screens/memory/memory_controller.dart:74]: will follow up in separate PR

Comment (polina-c) [reply packages/devtools_app/lib/src/screens/memory/memory_controller.dart:74]: https://github.com/flutter/devtools/pull/5071

**Pull request merged**